pkg/agent/cniserver/server.go

```diff
@@ -52,6 +53,8 @@ const (
 	// https://github.com/kubernetes/kubernetes/blob/v1.19.3/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L451
 	// networkReadyTimeout is set to a shorter time so it returns a clear message to the runtime.
 	networkReadyTimeout = 30 * time.Second
+	podAddComplete      = 1
```

Comment:
We should define these in cnipodcache/types.go.

Reply:
After reading the code, I really think we just need a boolean flag for CNIStatus like cniDeleted.